src/analysis/specific_limits.lean

```diff
@@ -695,6 +708,72 @@ end
 
 end normed_ring_geometric
 
+/-! ### Summability tests based on comparison with geometric series -/
+
+lemma summable_of_ratio_norm_eventually_le {α : Type*} [semi_normed_group α] [complete_space α]
+  {f : ℕ → α} {r : ℝ} (hr₀ : 0 ≤ r) (hr₁ : r < 1)
```

Comment:
I think this holds even without `hr₀`: if `r` is negative, then all `∥f n∥` are zero for large enough `n`, and the result is trivial (or, to say it differently, if the assumption holds for some `r` then it holds for `max 0 r`, so you can use the statement you have already proved).